backend/index.js

```diff
@@ -0,0 +1,32 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let pageVisitedTimesSession = 0;
+let pageVisitedTimesTotal = 0;
+
+try {
+  if (fs.existsSync("temp/my-counter-visited-times.txt")) {
+    (pageVisitedTimesTotal = Number(fs.readFileSync("temp/my-counter-visited-times.txt", "utf8")))
```

Comment:
Please remove extra braces 
```suggestion
    pageVisitedTimesTotal = Number(fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
```